.github/workflows/cmake.yml

```diff
@@ -10,28 +10,23 @@ jobs:
         include:
           - name: Ubuntu GCC
             os: ubuntu-latest
-            install_dir: ~/portaudio
             vcpkg_triplet: x64-linux
             cmake_generator: "Unix Makefiles"
             cmake_options:
               -DPA_USE_OSS=ON
-              -DCMAKE_TOOLCHAIN_FILE=vcpkg/scripts/buildsystems/vcpkg.cmake
           - name: Ubuntu MinGW
             os: ubuntu-latest
-            install_dir: ~/portaudio
             asio_sdk_cache_path: asiosdk.zip
             dependencies_extras: mingw-w64
-            # FIXME: linking JACK fails with vcpkg. Switch the CMake toolchain file to use vcpkg. The
-            # toolchain file in this repository is not needed when using a MinGW triplet with vcpkg.
             vcpkg_triplet: x64-mingw-static
             cmake_generator: "Unix Makefiles"
+            # FIXME: linking vcpkg Jack fais due to MinGW-W64 linker assertion. Disable Jack for this build.
             cmake_options:
               -DPA_USE_ASIO=ON
+              -DPA_USE_JACK=OFF
```

Comment:
Could you create a ticket regarding this issue please?

Reply:
There is linker error, so it is outside PA and either Jack dependency must be fixed or MinGW-64 linker has to be changed. I can create ticket to trace this issue although we could simply revise `DPA_USE_JACK=OFF` after some time.